win32ss/user/ntuser/security.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * PROJECT:         ReactOS Win32k subsystem
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:         Security infrastructure of NTUSER component of Win32k
+ * COPYRIGHT:       Copyright 2022 George Bișoc <george.bisoc@reactos.org>
+ */
+
+#pragma once
+
+//
+// USER objects security rights
+//
+
+/* Desktop access rights */
+#define DESKTOP_READ       STANDARD_RIGHTS_READ      | \
```

Comment:
For correctness it would be better to have opening and closing parentheses around the whole definition block:
`#define FOO (BAR1 | ... | BARN)`